src/main/kotlin/no/nav/syfo/MainApplication.kt

```diff
@@ -58,6 +60,10 @@ fun main() {
             val oversikthendelseRecordProducer = KafkaProducer<String, KOversikthendelse>(producerProperties)
             val oversikthendelseProducer = OversikthendelseProducer(oversikthendelseRecordProducer)
 
+            val oversikthendelseRetryProducerProperties = kafkaProducerConfig(env, vaultSecrets)
```

Comment:
Er det samme properties som for den vanlige kafka-topicen?

Reply:
Begge topics som det produserer til har samme properties akkurat nå ja.

Reply:
Trenger de da ulike variabler for properties?